imports/plugins/included/taxes-avalara/server/methods/taxCalc.js

```diff
@@ -15,11 +15,8 @@ const requiredFields = ["username", "password", "apiLoginId", "companyCode", "sh
 const taxCalc = {};
 
 taxCalc.getPackageData = function () {
-  const pkgData = Packages.findOne({
-    name: "taxes-avalara",
-    shopId: Reaction.getShopId(),
-    enabled: true
-  });
+  const pkgData = Reaction.getPackageSettings("taxes-avalara");
+  pkgData.settings.avalara = true;
```

Comment:
Help me understand why we need to set a value in the package settings?

Reply:
During my sync with @kieha , she made me aware the `enabled` boolean variable wasn't working properly and that `settings.avalara` is rather being used in the implementation of Avalara.

Reply:
Do we need to open an issue to make this work correctly if we are working around a bug elsewhere? We don't just hack something and move on, we would expect a `TODO` item at least.

Reply:
Duly noted, I would put a a `TODO` above it then